### DISCUSSION TITLE
Vue2 Documentation

### DISCUSSION THREAD

**Discussion by josejachuf** [quasarframework/quasar#9905] [Show and Tell]
Hi,

How do I access Quasar's documentation based on vue2? I'm sorry, but if you're in sight I'm not finding it. I only see the one of vue3?

Jose

**Comment by MilosPaunovic** [comment]
Quasar v1 documentation can be seen [here](https://v1.quasar.dev/).

**Comment by rstoenescu** [comment]
I think that you are referring to Quasar v1 (as opposed to v2 which is now the latest and greatest).
All versions are reachable from within our main docs website.

<img width="368" alt="older" src="https://user-images.githubusercontent.com/4665641/124155230-efee6c80-da9e-11eb-8fcd-c1126fa4f47b.png">

Quasar v1 points to https://v1.quasar.dev

**Reply by josejachuf** [reply]
Thanks
